### PR TITLE
sunos/*: add more information link

### PR DESCRIPTION
**Pull request by marchersimon**
For #5510

### CODE REVIEW

**Comment by marchersimon** [comment]
According to the urls `svcadm`, `svccfg`, `svcs` and `truss` are linux commands the rest is sunos only.

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Comment by sbrl** [comment]
> According to the urls `svcadm`, `svccfg`, `svcs` and `truss` are linux commands the rest is sunos only.

The copies in `sunos` are probably implemented slightly differently to the `linux` versions then I guess?

**Comment by marchersimon** [comment]
At least the man pages are the same. Is this reason enough to move them to common?

**Comment by bl-ue** [comment]
Seems like it to me. @sbrl?

File: pages.zh/sunos/dmesg.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # dmesg
 
 > 将内核消息写入标准输出.
+> 更多信息： <https://www.unix.com/man-page/sunos/1m/dmesg/>.
```
Comment (navarroaxel) [inline pages.zh/sunos/dmesg.md:4]:
```suggestion
> 更多信息： <https://www.unix.com/man-page/sunos/1m/dmesg>.
```

File: pages/sunos/devfsadm.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # devfsadm
 
 > Administration command for `/dev`. Maintains the `/dev` namespace.
+> More information: <https://www.unix.com/man-page/sunos/1m/devfsadm/>.
```
Comment (navarroaxel) [inline pages/sunos/devfsadm.md:4]:
```suggestion
> More information: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
```

File: pages/sunos/dmesg.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # dmesg
 
 > Write the kernel messages to standard output.
+> More information: <https://www.unix.com/man-page/sunos/1m/dmesg/>.
```
Comment (navarroaxel) [inline pages/sunos/dmesg.md:4]:
```suggestion
> More information: <https://www.unix.com/man-page/sunos/1m/dmesg>.
```

File: pages/sunos/prctl.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 > Get or set the resource controls of running processes,.
 > Tasks, and projects.
+> More information: <https://www.unix.com/man-page/sunos/1/prctl/>.
```
Comment (navarroaxel) [inline pages/sunos/prctl.md:5]:
```suggestion
> More information: <https://www.unix.com/man-page/sunos/1/prctl>.
```

File: pages/sunos/prstat.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # prstat
 
 > Report active process statistics.
+> More information: <https://www.unix.com/man-page/sunos/1m/prstat/>.
```
Comment (navarroaxel) [inline pages/sunos/prstat.md:4]:
```suggestion
> More information: <https://www.unix.com/man-page/sunos/1m/prstat>.
```

File: pages/sunos/svcadm.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 # svcadm
 
 > Manipulate service instances.
+> More information: <https://www.unix.com/man-page/linux/1m/svcadm/>.
```
Comment (navarroaxel) [inline pages/sunos/svcadm.md:4]:
```suggestion
> More information: <https://www.unix.com/man-page/linux/1m/svcadm>.
```

File: pages/sunos/snoop.md [hunk 7]
```diff
@@ -2,6 +2,7 @@
 
 > Network packet sniffer.
 > SunOS equivalent of tcpdump.
+> More information: <https://www.unix.com/man-page/sunos/1m/snoop/>.
```
Comment (navarroaxel) [inline pages/sunos/snoop.md:5]:
```suggestion
> More information: <https://www.unix.com/man-page/sunos/1m/snoop>.
```

File: pages/sunos/svccfg.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 # svccfg
 
 > Import, export, and modify service configurations.
+> More information: <https://www.unix.com/man-page/linux/1m/svccfg/>.
```
Comment (navarroaxel) [inline pages/sunos/svccfg.md:4]:
```suggestion
> More information: <https://www.unix.com/man-page/linux/1m/svccfg>.
```

File: pages/sunos/svcs.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 # svcs
 
 > List information about running services.
+> More information: <https://www.unix.com/man-page/linux/1/svcs/>.
```
Comment (navarroaxel) [inline pages/sunos/svcs.md:4]:
```suggestion
> More information: <https://www.unix.com/man-page/linux/1/svcs>.
```

File: pages/sunos/truss.md [hunk 10]
```diff
@@ -2,6 +2,7 @@
 
 > Troubleshooting tool for tracing system calls.
 > SunOS equivalent of strace.
+> More information: <https://www.unix.com/man-page/linux/1/truss/>.
```
Comment (navarroaxel) [inline pages/sunos/truss.md:5]:
```suggestion
> More information: <https://www.unix.com/man-page/linux/1/truss>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**